test.zig

```diff
@@ -62,8 +65,8 @@ pub fn main() !u8 {
         try tests.append(Test.init(&[_][]const u8 { ziget, "google.com"}));
         try tests.append(Test.init(&[_][]const u8 { ziget, "http://google.com"}));
     }
-    if (openssl) {
```

Comment:
Do you want to keep the regular openssl too? Or is it going away in favor of the static one?

Reply:
Is there any reason to remove it?

Reply:
No, but you removed it in the tests, that is why I am asking.

Reply:
woops